### PR TITLE
The Project Section of the portfolio design

### PR DESCRIPTION
**Pull request by sainingo**
- I've used the grid to display the sections on the web page
- Then used flex to align the children on the page
- I've added hover effects on the buttons

## Changes requested:
Fix horizontal scroll - Some elements had big widths had to resolve that
Projects section to have a display of grid  > fixed
And lastly, change the div inside projects to be ul and li. > fixed

_Please let me know if there's anything i can add or change_

**N/B:**  No functionalities added on the page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -90,3 +90,406 @@ header nav a {
   list-style: none;
   color: inherit;
 }
+
+.projects {
+  margin: 0 20px;
+  position: relative;
```
Comment (AnselemOdims) [inline styles.css:96]: - [ ] Kindly layout the elements in this section using grid `display: grid`

File: index.html [hunk 2]
```diff
@@ -40,6 +40,181 @@ <h1>Hey there. I'm Sainingo <br><span>I'm a software developer</span></h1>
...
+            <h4>About Me</h4>
+            <p>I can help you build a product, feature or website look through some of my work and experience!
+                if you like what you see and have a project you need coded, dont hesitate to contact me.
+            </p>
+            <button type="button">Get My Resume</button>
+         </div>
+         <!-- cards -->
+         <div class="card-section">
+             <img src="./assets/images/icon-lang.png" alt="image placeholder">
+             <p>Languages</p>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

File: index.html [hunk 3]
```diff
@@ -40,6 +40,181 @@ <h1>Hey there. I'm Sainingo <br><span>I'm a software developer</span></h1>
...
+            <p>Skills</p>
+            <ul class="card-2">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+     </section>
```
Comment (AnselemOdims) [inline index.html]: - [ ] There seems to be a vertical scroll on the page. Kindly remove or prevent the scroll as it affects the user experience.

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @sainingo , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. Kindly tag me using @AnselemOdims for a quicker reply

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (sainingo) [reply index.html]: Hi @AnselemOdims. Thanks for taking your time to review my code, Kindly am not understanding that this part? please clarify for me

File: styles.css (hunk 1 above)
Comment (sainingo) [reply styles.css:96]: Thanks I will sure implement that right away

File: index.html (hunk 2 above)
Comment (sainingo) [reply index.html]: @AnselemOdims still waiting??

Comment (sainingo) [reply index.html]: Hi @AnselemOdims

File: index.html (hunk 3 above)
Comment (sainingo) [reply index.html]: Thanks was able to fix

File: index.html (hunk 2 above)
Comment (AnselemOdims) [reply index.html]: Hi @sainingo, 
This means the languages, frameworks and skills should also be as list items in a list wrapper.
```
<ul>
  <li>Languages</li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```

File: styles.css (hunk 1 above)
Comment (Whoistolu) [reply styles.css:96]: 👍🏽 Good now.

File: index.html (hunk 3 above)
Comment (Whoistolu) [reply index.html]: 👍🏽 Good now.

File: index.html (hunk 2 above)
Comment (Whoistolu) [reply index.html]: 👍🏽 Good now.

File: index.html [hunk 4]
```diff
@@ -40,6 +41,185 @@ <h1>Hey there. I'm Sainingo <br><span>I'm a software developer</span></h1>
...
-        </main>
+     </main>
+
+ 
+     <section class="projects">
+        <div class="project-flex">
+            <span class="disable-icon"><img src="./assets/images/Disabled.png" alt="disable" ></span>
+            <h2>My Recent Works</h2>
+            <span><img src="./assets/images/Indicator.png" alt="indicator"></span>
+        </div>
```
Comment (Whoistolu) [inline index.html:52]: - [ ] [OPTIONAL] Just above "My Recent Works", there is a big white space that should not be in the design. Kindly remove it.
Screenshot from 2022-02-16 10-48-54

**Review by Whoistolu — APPROVED** [review]
### Hi @sainingo,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (sainingo) [reply index.html:52]: Ok. Thanks for the feedback

**Pull request merged**